app/helpers/application_helper.rb

```diff
@@ -151,4 +151,12 @@ def event_description(event)
   def can_ask_the_eu?(code)
     WorldFOIWebsites.can_ask_the_eu?(code)
   end
+
+  def controller?(*controller)
```

Comment:
Maybe these params should be *controller**s** and *action**s** just to be totally clear?